invenio_records_marc21/services/permissions.py

```diff
@@ -174,3 +174,29 @@ class Marc21RecordPermissionPolicy(RecordPermissionPolicy):
     #
     # Miscellaneous
     can_query_stats = [Disable()]
+
+
+class LenientMarc21PermissionPolicy(Marc21RecordPermissionPolicy):
```

Comment:
if you need that for invenio-overrides then this should be moved to `invenio-overrides`. for the package `invenio-records-marc21` it is enough to have one permission policy. adding a new one could lead to confusions for users of the packages and for developers. for developers it could be difficult to decide how and where to add changes.

Reply:
It's not about the invenio-overrides, but also in the repository as shown its false checking there when anonymous users. That should be done in the package itself than with a workaround in invenio-override. 

Sure, if it's confusing, we can handle it that way too.

Reply:
but the fix is the change in the records.html file (or if moved to files.html file then there) but adding a new permission policy doesn't fix anything, because it is not used

Reply:
yes but my idea was while the template changes fix the UI display, the lenient policy approach is better because it solves the actual permission errors in the repository layer while giving the flexibility to choose between strict and lenient behaviors, with the original policy remaining as the default.

Reply:
what do you mean by `actual permission error`?

Reply:
'errors' - three cases as mention in the PR.